packages/dso-toolkit/src/components/legend-item/legend-item.models.ts

```diff
@@ -0,0 +1,14 @@
+import { Selectable } from "../selectable/selectable.models";
+
+export interface LegendItem<TemplateFnReturnType> {
+  body?: TemplateFnReturnType;
+  disabled?: boolean;
+  disabledMessage?: string;
+  dsoMouseEnter?: () => void;
+  dsoMouseLeave?: () => void;
+  dsoRemoveClick?: (e: CustomEvent<MouseEvent>) => void;
+  label: string;
```

Comment:
Functioneel gezien is label optioneel, als ik een eerder commentaar goed begrijp. Deze property zou dat dan ook moeten zijn, correct?